app/assets/stylesheets/widgets/home_page/emergency_alert.scss

```diff
@@ -0,0 +1,105 @@
+// Global value for large-screen is currently 1024px
+// Overwrite just in this file
+$large-screen: 1130px;
+
+.emergency-alert {
+  display: none;
+  background-color: $cu-red;
+  color: $white;
+  padding: 25px;
+  border-bottom: 1px solid $cu-light-grey;
+
+  .close-alert-trigger {
+    cursor: pointer;
+    margin: 5px;
+    float: right;
+
+    svg {
+      margin: 0;
+    }
+
+    @include media($medium-screen) {
+      margin: 10px;
+    }
+
+    @include media($large-screen) {
+      margin: 15px;
+    }
+  }
+
+  .alert-text {
+    @include media($medium-screen) {
+      padding: 0 60px;
+    }
+
+    @include media($large-screen) {
+      padding: 0 250px;
+    }
+
+    p.alert-heading {
+      text-transform: uppercase;
+      font-family: $font-stack-heading;
+      font-size: 24px;
+      font-weight: 800;
+      text-align: center;
+      padding-bottom: 5px;
+
+      @include media($medium-screen) {
+        text-align: left;
+        display: inline-block;
+        font-size: 26px;
+      }
+    }
+
+    div {
+      font-family: $font-stack-body;
+      font-weight: 400;
+      font-size: 16px;
+      line-height: 22px;
+
+      &.alert-title {
+        padding-top: 20px;
+        font-weight: 600;
+        font-size: 18px;
+        border-top: 1px solid white;
+      }
+
+      &.alert-message {
+        padding-top: 10px;
+        padding-bottom: 20px;
+        border-bottom: 1px solid white;
+
+        // Styles for potential WYSIWYG content (data def in Cascade)
+        p {
+          font-size: 16px;
+          line-height: 22px;
+        }
+
+        h1, h2, h3 {
```

Comment:
not sure why we'd have the same size for h1, h2 and h3, or why we'd ever anticipate an h1 in emergency alert (would be invalid I think). Just curious

Reply:
This is kind of hard to explain. The design in the card allows for a title, which is automatically pulled from Rave and inserted into `<div class="alert-title">`. _Unless_ Ross (or whoever) enters a manual message in the data definition, which overrides everything from the feed. They would use the WYSIWYG for that, and all of the copy they put in the WYSIWYG gets put inside the general message div in the HTML and not the alert-title div, so it doesn't get the right styles, it won't look the way the design does (see trello). So in order to kind of force the design, I added these rules that just accommodate the different heading tags that Ross (or whoever) might use inside the WYSIWYG. I can change this CSS rule to just one heading level, but there's the possibility that someone will use a different heading level, which will end up looking pretty bad on the front end (I checked), just because of the default heading styles we have in place.

Up to you.